.github/workflows/docker-image.yml

```diff
@@ -2,13 +2,18 @@ name: Publish Docker Image to Docker Hub
 
 on:
   workflow_dispatch:
+    inputs:
```

Comment:
Add an optional input to specify the version tag. This way we can test images like "0.0.138-dimensionFilter-test" without publishing partial updates on the main version line.
This can thenbe use for the Credible Worker image tag to test new worker capabilities.